internal/ingest/ingest_test.go

```diff
@@ -21,24 +21,40 @@ import (
 	persistence_fake "github.com/artefactual-sdps/enduro/internal/persistence/fake"
 )
 
-func testSvc(t *testing.T, b *blob.Bucket, s int64) (ingest.Service, *persistence_fake.MockService) {
+// io.Reader used as number generator for making UUIDs predictable
+type staticRand struct{}
```

Comment:
I think we usually use `rand.New(rand.NewSource(1))` (or similar) for a deterministic number generator. Is there a reason to create your own number generator here?